packages/@lwc/synthetic-shadow/src/faux-shadow/html-element.ts

```diff
@@ -170,3 +204,29 @@ defineProperties(HTMLElement.prototype, {
         configurable: true,
     },
 });
+
+// Note: Firefox does not have outerText, https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/outerText
+if (outerTextGetter !== null && outerTextSetter !== null) {
+    defineProperty(HTMLElement.prototype, 'outerText', {
+        get(this: Element): string {
+            if (!featureFlags.ENABLE_ELEMENT_PATCH) {
+                if (isNodeShadowed(this) || isHostElement(this)) {
+                    return getInnerText.call(this);
+                }
+
+                return outerTextGetter!.call(this);
+            }
+
+            // TODO [#1222]: remove global bypass
+            if (isGlobalPatchingSkipped(this)) {
+                return outerTextGetter!.call(this);
+            }
+            return getInnerText.call(this);
+        },
+        set(v: string) {
+            outerTextSetter!.call(this, v);
```

Comment:
It's an edge case but invoking the `outerText` setter on a host element should trigger its disconnection. Until we merge node reaction, it will not work. 

We can always reimplement the outerText setter in JavaScript ([blink implementation](https://source.chromium.org/chromium/chromium/src/+/master:third_party/blink/renderer/core/html/html_element.cc;l=841-879;drc=6e8b402a6231405b753919029c9027404325ea00;bpv=0;bpt=1)), but I think it will be a waste of time.

Reply:
Can we add a comment for this?

Reply:
done